api/services/elastigroup/gcp/paths/group.yaml

```diff
@@ -41,14 +39,21 @@ put:
     - "Elastigroup GCP"
   parameters:
     - $ref: "../../../../commons/parameters/accountId.yaml"
-    - $ref: "../../commons/parameters/groupId.yaml"
+    - in: "path"
```

Comment:
@vlad666spot please use the common param

Reply:
fixed